cluster-autoscaler/cloudprovider/mcm/mcm_manager.go

```diff
@@ -722,6 +757,28 @@ func (m *McmManager) GetMachineDeploymentNodeTemplate(machinedeployment *Machine
 	return nodeTmpl, nil
 }
 
+func filterOutNodesWithCapacity(nodes []*v1.Node) []*v1.Node {
+	var filteredNodes []*v1.Node
+
+	for _, node := range nodes {
+		if node.Status.Capacity != nil {
+			filteredNodes = append(filteredNodes, node)
+		}
+	}
+
+	return filteredNodes
+}
+
+func getWorkerPoolForMachineDeploy(md *v1alpha1.MachineDeployment) string {
+	if md.Spec.Template.Spec.NodeTemplateSpec.Labels != nil {
+		if value, exists := md.Spec.Template.Spec.NodeTemplateSpec.Labels[nodegroupset.LabelWorkerPool]; exists {
```

Comment:
`value, ok` is sort of a standard practice in Go as I have seen in several codebases. How about we stick to it too?

Reply:
sure.